sre_maturity.md

```diff
@@ -0,0 +1,115 @@
+# SRE Maturity Model
+
+## Phase 1: The ‘Crawl’ Phase
+
+### Overview
+
+You’re starting from zero, with a net new team.
+You may only have a small SRE team, or an SRE function within your development team.
+You and your team are technically capable of surviving.
+You are probably providing a ‘white glove’ experience to your users at this stage.
+
+### Characteristics
+
+#### People and Culture
+
+You have a strong SRE voice in your team, someone who advocates for reliability features.
+They establish people focused mantras like "It's OK to fail", "Hope is not a strategy" and "Trust by default & extend on it".
+Communication is important in your team and you hire diverse people based on soft skills & leadership rather than specific technologies.
+
+At this phase, having buy-in from executive level leadership is essential.
+Without this it will be very difficult to get their support later, and eventually their engagement.
+
+The product manager for you service should have a good understanding of SRE.
+They should help you advocate for service features that make the service more reliable.
+
+#### Observability
```

Comment:
@jeremyeder @mmazur @lisa 
Probably something for future PRs, but can I get your thoughts on a 'Pitfalls' section in each phase.
It would highlight lessons learned where we've tried something and it didn't work or it slowed things down.

For example, in kafka, a lot of alerts were defined very early in the service journey. People familiar with kafka & strimzi came up with lots of great ways to detect that something may be wrong by querying various metrics. 
This led to a few pain points.
* lots of alerts being lowered to 'warning' severity, firing regularly in production and essentially being ignored. (A good alerting consistency model & policy can help with this)
* The sheer volume of alerts tied to a hard requirement that came later to add SOPs for all alerts (why have an alert that isn't actionable?) led to a big slowdown while playing catchup.

There's another similar pain point where alerts defined in an upstream project/product are taken as is. This can lead to false positives as thresholds may be different or the alert use case is a little different than how you're using the upstream project/product.

Reply:
Maybe each phase has a section on observability and for the "Crawl phase" one characteristic is an excess of alerts that are later silenced, ignored, or have thresholds tuned to page less often. Later phases can refine the practice by pruning "vanity alerts" and honing in on the alerts which directly support specific SLOs.

Reply:
That makes sense @lisa 
I'm going to rework the model to support this approach.